include/RAJA/index/IndexSet.hpp

```diff
@@ -500,7 +497,7 @@ class StaticIndexSet<T0, TREST...> : public StaticIndexSet<TREST...>
   {
     StaticIndexSet<T0, TREST...> *retVal = new StaticIndexSet<T0, TREST...>();
     int numSeg = getNumSegments();
-    for (auto & seg : segIds)
+    for (auto &seg : segIds)
```

Comment:
Can we add some curly braces to this like this to avoid logic/conceptual errors/misunderstanding?

Reply:
Sure, I have no issue with that.  Actually I rather wish clang-format had an option to add them for us, but sadly it does not. 😞

Reply:
Also, it appears that most of the changes in the PR involve formatting so it's kinda hard to tell what's really changed. I think we need to get to a point where no code would change in develop by running it through our formatting tool. Then, we need to require all new code to be properly formatted.  Agree?

Reply:
Another thing we may consider is that when we switch over to BLT (which I assume we will do at this point), we would have uncrustify for free.  That would give us more formatting capabilities.